src/main/java/ro/utcn/sd/boti/stackoverflow/entity/Question.java

```diff
@@ -13,7 +13,7 @@
 @Entity
 @NoArgsConstructor
 @AllArgsConstructor
-@ToString(of = {"id", "title", "text", "time", "vote"})
+//@ToString(of = {"id", "title", "text", "time", "vote"})
```

Comment:
Clean-up commented out code. That's what git is for: you can look at the git history instead of keeping dead commented code inside your code base.